make/include/versioning

```diff
@@ -25,7 +25,8 @@ GIT_TAG=${GIT_TAG:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $1 }' )}
 GIT_COMMITS=${GIT_COMMITS:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $2 }' )}
 GIT_SHA=${GIT_SHA:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $3 }' )}
 
-CF_VERSION=$(cat "${GIT_ROOT}/CF_VERSION")
+. $(dirname $(dirname "$0"))/bin/dev/versions.sh
```

Comment:
So I suggest to switch to

```
. $(dirname $(dirname $(dirname "$BASH_SOURCE")))/bin/dev/versions.sh
```

to become independent of the location of the caller and only rely on the location of this script itself.